sklearn/discriminant_analysis.py

```diff
@@ -623,11 +623,28 @@ class QuadraticDiscriminantAnalysis(BaseEstimator, ClassifierMixin):
         Discriminant Analysis
     """
 
-    def __init__(self, priors=None, reg_param=0., store_covariances=False,
-                 tol=1.0e-4):
+<<<<<<< HEAD
```

Comment:
Everything looks fine apart from the fact that your merge wasn't very clean. Remove lines:

- 626
- 637-647
- 721
- 723-725